tools/hyphy/examples/v.1.0.11.html

```diff
@@ -0,0 +1,18 @@
+<script src="https://biowasm.com/cdn/v3/aioli.js"></script>
```

Comment:
You can delete those 2 html files and make one called `tools/hyphy/examples/2.5.52.html` with an example HyPhy command.

Then you can modify `web/tests/test_cdn.cy.js` to add a line for hyphy/2.5.52 and which output you expect to see when running that command

Reply:
It's completed, but I'm unsure how to test the page with Aioli. I did make sure that the compiled version of hyphy ran `help` and `version` correctly.